src/algebra/group/hom.lean

```diff
@@ -138,56 +216,117 @@ lemma map_exists_left_inv (f : M →* N) {x : M} (hx : ∃ y, y * x = 1) :
   ∃ y, y * f x = 1 :=
 let ⟨y, hy⟩ := hx in ⟨f y, f.map_mul_eq_one hy⟩
 
-omit mN mM
+end monoid_hom
 
+/-- The identity map from a type with 1 to itself. -/
+@[to_additive]
+def one_hom.id (M : Type*) [has_one M] : one_hom M M :=
+{ to_fun := id,
+  map_one' := rfl, }
```

Comment:
I'm usually not too concerned about adding an empty line between each declaration, but for multi-line declarations it definitely helps.
```suggestion
  map_one' := rfl, }

```

Reply:
Alternatively, should I just collapse the structure into one line? I wanted to try and keep the clusters of identical defs obvious.

Reply:
Either way is fine by me.

Reply:
I think for such a short structure, that is a good compromise.